app/routes.py

```diff
@@ -1,2 +1,239 @@
-from flask import Blueprint
+from flask import Blueprint, jsonify, make_response,request
+from app.models.task import Task
+from app.models.goal import Goal
+from app import db
+from datetime import datetime 
+#what comes from the "from" is the package name
+#what comes after the import is a variable that matches the name of thing we want to import
+tasks_bp = Blueprint("tasks", __name__,url_prefix=("/tasks"))
+
+goals_bp = Blueprint("goals", __name__, url_prefix=("/goals"))
+
+@goals_bp.route("",methods=["POST", "GET"])
+def handle_goal():
+      if request.method == "POST":
+        request_body = request.get_json()
+        if "title" not in request_body:
+            return{
+                "details": "Invalid data"
+            },400
+
+        new_goal = Goal (
+            title=request_body["title"]
+        )
+
+        db.session.add(new_goal)
+        db.session.commit()
+
+        return {
+            "goal": {
+                "id":new_goal.goal_id,
+                "title":new_goal.title
+            }
```

Comment:
Don't forget about you `goal_dict` function. You should be able to use that here.